lib/bundler/rubygems_gem_installer.rb

```diff
@@ -31,12 +31,24 @@ def validate_bundler_checksum(checksum)
         send(checksum_type(checksum), digest)
       end
       unless digest == checksum
-        raise SecurityError,
-          "The checksum for the downloaded `#{spec.full_name}.gem` did not match " \
-          "the checksum given by the API. This means that the contents of the " \
-          "gem appear to be different from what was uploaded, and could be an indicator of a security issue.\n" \
-          "(The expected SHA256 checksum was #{checksum.inspect}, but the checksum for the downloaded gem was #{digest.inspect}.)\n" \
-          "Bundler cannot continue installing #{spec.name} (#{spec.version})."
+        raise SecurityError, <<-MESSAGE
```

Comment:
Are you suggesting to remove all the indents for lines 35 to 51 or just perform a `lstrip` on the entire string?